src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.ServiceConnection.json

```diff
@@ -41,13 +41,13 @@
       "Enabled": true
     },
     {
-      "ControlID": "ADO_ServiceConnection_AuthZ_Justify_Connection_Admin",
+      "ControlID": "ADO_ServiceConnection_AuthZ_Grant_Min_RBAC_Access",
       "Description": "Justify all users/groups that have access to the service connection.",
       "Id": "ServiceConnection130",
       "ControlSeverity": "High",
-      "Automated": "No",
-      "MethodName": "CheckServiceConnectionAdmins",
-      "Rationale": "Accounts with admin access can install/manage extensions for Organization. Members with this access without a legitimate business reason increase the risk for Organization. By carefully reviewing and removing accounts that shouldn't be there in the first place, you can avoid attacks if those accounts are compromised.",
+      "Automated": "Yes",
+      "MethodName": "CheckRBACAccess",
+      "Rationale": "Granting minimum access by leveraging RBAC feature ensures that users are granted just enough permissions to perform their tasks. This minimizes exposure of the resources in case of user/service account compromise.",
```

Comment:
"Granting minimum access by leveraging RBAC feature ensures that users/groups are granted just enough permissions on service connection to perform their tasks. This minimizes exposure of the resources in case of user/service account compromise.